python-package/lightgbm/dask.py

```diff
@@ -319,6 +291,8 @@ def _train_part(
         if eval_class_weight:
             kwargs['eval_class_weight'] = [eval_class_weight[i] for i in eval_component_idx]
 
+    if remote_socket is not None:
+        remote_socket.release()
```

Comment:
very very very minor comment... but would you consider moving this block down a line, to after `model = model_factory(**params)`?

The `Network` isn't initialized until `Booster.__init__()`

https://github.com/microsoft/LightGBM/blob/07e3cf47f6a01cc6f7692a70ff5288eee43ce12d/python-package/lightgbm/basic.py#L3085-L3090

which isn't called until `{estimator}.fiit(...)`

https://github.com/microsoft/LightGBM/blob/07e3cf47f6a01cc6f7692a70ff5288eee43ce12d/python-package/lightgbm/sklearn.py#L839-L839

Switching the order would further reduce the risk of some other process claiming that port between when the socket is released and when LightGBM claims it during `{estimator}.fit()`.

Reply:
Changed in 9c7cabb

Reply:
thanks!